src/objects/creep_shared.rs

```diff
@@ -338,15 +404,15 @@ where
     }
 
     /// Sets cost callback - default `|_, _| {}`.
-    pub fn cost_callback<'b, F2>(self, cost_callback: F2) -> MoveToOptions<'b, F2>
+    pub fn cost_callback<'b: 'a, F2>(self, cost_callback: F2) -> MoveToOptions<'a, F2>
```

Comment:
Looks like there's a lifetime change included here that isn't used anywhere. Is this intentional, or could it be put back?

It looks like the kind of thing you'd need if you stored borrowed strings in PolyStyle - which is valid, but since we aren't doing it I don't _think_ it should be necessary.